commands-yml/commands/session/events/get-events.yml

```diff
@@ -35,9 +35,7 @@ example_usage:
 client_docs:
   java:
     - url: "https://seleniumhq.github.io/selenium/docs/api/java/org/openqa/selenium/WebElement.html#click--"
-  python:
-    - android: "https://github.com/appium/python-client/android-command"
-      ios: "https://github.com/appium/python-client/ios-command"
+  python: "https://github.com/appium/python-client/blob/master/appium/webdriver/extensions/log_event.py#L22"
```

Comment:
I'd rather simply put a link to the file. Line numbers might be a subject of change in the future

Reply:
same below